charts/gke-operator/values.yaml

```diff
@@ -18,3 +18,5 @@ nodeSelector: {}
 ## List of node taints to tolerate (requires Kubernetes >= 1.6)
 tolerations: []
 
+## PriorityClassName assigned to deployment.
+priorityClassName: ""
```

Comment:
So does this get set via the rancher installer ? It wouldn't come from helm.

Reply:
Yes, Rancher Manager will install the gke-operator with the prioritClassName value set. We are defaulting here to an empty string to maintain backward compatibility.